src/kmod.rs

```diff
@@ -0,0 +1,108 @@
+//! Load and unload kernel modules.
+
+use libc;
+use std::ffi::CStr;
+use std::os::unix::io::AsRawFd;
+
+use errno::Errno;
+use Result;
+
+/// Loads an ELF image into kernel space, performs any necessary symbol relocations,
```

Comment:
Please use a short (60 char or less) title sentence, followed by a blank comment line, then the rest of the text. See how other functions are commented for examples.